.changeset/swift-dogs-care.md

```diff
@@ -0,0 +1,6 @@
+---
+'create-pantheon-decoupled-kit': minor
+---
+
+[next-drupal] User entered search term persists on search results page in the
```

Comment:
```suggestion
[next-drupal-search-api-addon] User entered search term persists on search results page in the
```
More specificity with the tag here so we can easily tell which generators were changed at a glance of the changelog